### DISCUSSION TITLE
Failed modified kafkatopic after kafka cluster restart

### DISCUSSION THREAD

**Discussion by northboys** [strimzi/strimzi-kafka-operator#6860] [Q&A]
my kafka cluster restart after that i try to modify kafkatopic but got error like this.
```
Failure processing KafkaTopic watch event MODIFIED on resource mytopic with labels {strimzi.io/cluster=my-cluster}: The state store, topic-store, may have migrated to another instance.
```
and i try to create new topic and status topic `NotReady `.

all kafkacluster pod running fine.

**Comment by northboys** [comment]
Solved by restart entity-operator 😁

**Reply by AydinChavez** [reply]
how did you restart the entity-operator? i am asking because k8s does not provide a restart functionality out of the box.

**Reply by scholzj** [reply]
You can just delete the pod with `kubectl delete`.

**Reply by AydinChavez** [reply]
thx, this is what I did. but than the topic operator  would not start, same exceptions as here: https://github.com/strimzi/strimzi-kafka-operator/issues/6046
therefore i adjusted the template as adjusted there (STRIMZI_USE_ZOOKEEPER_TOPIC_STORE=true) and also increased the mem+cpu for topic and useroperator like described here https://github.com/strimzi/strimzi-kafka-operator/issues/1050. let's see if it helps.

**Reply by AydinChavez** [reply]
ok, seems to work so far.
